### ISSUE TITLE
Code in readme is wrong

### ISSUE THREAD

**Issue by VVZen** [GoogleCloudPlatform/cloud-datastore-session-node#4]
It is:

```js
app.use(session({
  store: new DatastoreStore({
    dataset: gcloud.datastore({
        prefix: 'express-sessions',
        projectId: 'my-project-id',
        keyFilename: './key.json'
    }))
  }),
  secret: 'my-secret'
}));

```

should be:

```js
app.use(session({
  store: new DatastoreStore(
  // session store object
  {
    dataset: gcloud.datastore({
      prefix: "express-sessions",
      projectId: "my-project-id",
      keyFilename: "./key.json"
    })
  }),
  secret: "my-secret"
}));
```

There's one too many parenthesis after _keyFilename_.
PS: Is this repository **still alive**?

**Comment by tswast** [comment]
@jmdobry Have these samples moved to https://github.com/GoogleCloudPlatform/nodejs-docs-samples ?

**Comment by VVZen** [comment]
@tswast I did a quick search in **nodejs-docs-samples** but I didn't find anything documenting the usage of the package _google-datastore-session_ for saving user sessions on Google Cloud Datastore. 🤔

**Comment by jmdobry** [comment]
@tswast This isn't a sample, it's a library for using Cloud Datastore as your backing store for user sessions in Express.js apps.

I wasn't aware of this repo before now, but now that I am, I'll get everything updated.

**Comment by jmdobry** [comment]
This should be fixed in 1.0.0. Note that the package has been renamed to `@google-cloud/connect-datastore` on NPM.
